src/ia-book-search-results.js

```diff
@@ -66,6 +66,22 @@ export class IABookSearchResults extends LitElement {
     this.query = e.currentTarget.value;
   }
 
+  updateSearchState(e) {
+    this.setQuery(e);
+
+    if (!this.query) {
+      this.emitSearchCancelled();
+    }
+  }
+
+  emitSearchCancelled() {
+    this.dispatchEvent(new CustomEvent('bookSearchResultsCleared', {
+      detail: {
+        query: this.query,
```

Comment:
Since we are calling this event when `!this.query`, can we just set empty value ('') instead of write variable name as its already blank.